src/flb_config.c

```diff
@@ -232,6 +232,11 @@ struct flb_config *flb_config_init()
 
     /* Set default coroutines stack size */
     config->coro_stack_size = FLB_CORO_STACK_SIZE;
+    if (config->coro_stack_size < getpagesize()) {
+        flb_info("[config] change coro_stack_size %u -> %u bytes",
```

Comment:
suggestion: use slightly different wording `changing coro_stack_size from %u to %u bytes`

Reply:
Thank you.
I modified the log message.